examples/app.js

```diff
@@ -76,3 +76,8 @@ export async function showSignedTransactions() {
 
     alert(JSON.stringify(plainSignedTransactions, null, 4));
 }
+
+export async function logout() {
+    let provider = createProvider();
+    await provider.logout();
```

Comment:
Let's add a callbackUrl here, to show that we need that property for this to work properly